templates/main/multipassage.html

```diff
@@ -65,14 +86,21 @@ <h3 class="entry-title text-center">{{object.collections.current.title}}<a id="{
     <div class="">
     <!-- The Modal container for the eLexicon entries. -->
     <div class="modal" id="lexicon-modal" message="{{ _('Es gibt derzeit noch keinen Lexikoneintrag zu') }} " tabindex="-1" role="dialog" aria-labelledby="lexicon-modalLabel" aria-hidden="true">
-    
+
     </div>
         <article class="hentry entry white col-sm-12">
             <!-- <nav>
                 {{ navigation }}
             </nav> -->
-            
+
             <div class="row">
+                <section class="col-sm with-border" id="viewer" style="display:none;">
+
+
+
+
+
```

Comment:
I would delete all of these blank lines and put the closing `</section>` tag on the same line as the opening tag.